src/main/webapp/app/core/about-us/about-us.component.ts

```diff
@@ -17,15 +17,42 @@ export class AboutUsComponent implements OnInit {
     private readonly issueBaseUrl = 'https://github.com/ls1intum/Artemis/issues/new?projects=ls1intum/1';
     readonly bugReportUrl = `${this.issueBaseUrl}&labels=bug&template=bug-report.yml`;
     readonly featureRequestUrl = `${this.issueBaseUrl}&labels=feature&template=feature-request.yml`;
-    readonly examFeatureOverviewUrlStudents = '../features/students/';
-    readonly examFeatureOverviewUrlInstructors = '../features/instructors/';
-    readonly examInstructionsUrlStudents = 'https://docs.artemis.ase.in.tum.de/user/exams/students_guide/';
-    readonly examInstructionsUrlInstructors = 'https://docs.artemis.ase.in.tum.de/user/exams/instructors_guide/';
     readonly releaseNotesUrl = `https://github.com/ls1intum/Artemis/releases/tag/${VERSION}`;
 
     email: string;
     data: AboutUsModel;
 
+    // Array of tuple containing list of details of section
+    readonly numberOfDetails: [string, number, { [key: string]: string }][] = [
+        [
+            'exercises.programming',
+            8,
+            {
+                programmingUrl: 'https://docs.artemis.ase.in.tum.de/user/exercises/programming/',
+                orionUrl: 'https://github.com/ls1intum/Orion',
+                aresUrl: 'https://github.com/ls1intum/Ares',
+            },
+        ],
+        ['exercises.quiz', 2, { quizUrl: 'https://docs.artemis.ase.in.tum.de/user/exercises/quiz/' }],
+        ['exercises.modeling', 3, { modelingUrl: 'https://docs.artemis.ase.in.tum.de/user/exercises/modeling/', apollonUrl: 'https://apollon.ase.in.tum.de/' }],
+        ['exercises.text', 2, { textUrl: 'https://docs.artemis.ase.in.tum.de/user/exercises/textual/', athenaUrl: 'https://github.com/ls1intum/Athena' }],
+        ['exercises.fileUpload', 0, { fileUploadUrl: 'https://docs.artemis.ase.in.tum.de/user/exercises/file-upload/', athenaUrl: 'https://github.com/ls1intum/Athena' }],
+        ['exam', 2, { studentFeatureUrl: 'https://artemis.ase.in.tum.de/features/students', instructorFeatureUrl: 'https://artemis.ase.in.tum.de/features/instructors' }],
```

Comment:
Before the rework, the page here used relative links to point to the feature overview of the current artemis instance (instead of always to the TUM-Instance). If you for example compare https://artemis.cs.hm.edu/features/students with https://artemis.ase.in.tum.de/features/students you'll note some differences with e.g "LRZ Login" only being a feature of our Artemis instance.

Reply:
Valid point - yes, you are right :+1: